paper/writeup.tex

```diff
@@ -1159,32 +1186,38 @@ \section{Discussion}
 of the true ancestry of a set of samples (i.e., all their genetic ancestors who ever lived),
 and so we wanted to quantify
 ``How much of the true relationships does this ARG represent?''
-However, the symmetrized version
+However, it is worth noting that the symmetrized version
 $d(\T_1,\T_2) = \dis(\T_1,\T_2) + \dis(\T_2, \T_1)$ is a metric.
 To see this, first suppose that we have a bijection between the nodes of $\T_1$ and $\T_2$,
 and view each ARG as a subset of the space $[0,L) \times N \times N$,
 where $N$ is the shared set of nodes.
-Then, dissimilarity is the Lesbegue measure of the relative difference of the two sets:
+Then, dissimilarity is the Lebesgue measure of the relative difference of the two sets:
 $|\T_1 \setminus \T_2|$,
 and so the symmetrized version is the measure of the symmetric difference
 $|\T_1\Delta\T_2|$,
-which is well-known to be a metric.
+which is well-known to be a metric. % NSP-22NOV: "well-known" ==> give a citation here
+% HF-24NOV: Peter would we cite Rudin?
```

Comment:
I don't know - have you looked in Rubin? (But - I agree a citation would be nice if we have one handy, but it's not totally necessary.) It's [in wikipedia](https://en.wikipedia.org/wiki/Symmetric_difference#Symmetric_difference_on_measure_spaces) after all?